CONTRIBUTING.md

```diff
@@ -23,15 +23,16 @@ We will then:
 
 To speed up the review process, please:
 
-- write your code cleanly. We also have our day time work, we are not able to correct your code for mistakes
-- describe cleanly what your pull request does
+- write your code cleanly. We also have our day time work, we are not able to correct the mistakes in your code
+- describe clearly what your pull request does
 - if you're fixing a particular bug in the issue list, please explicitly say "Fixes #<issue number>" in your description
 - while not required (because automated tests can't test everything), it's recommended you to include [Robolectric](http://robolectric.org/)/[Espresso](https://developer.android.com/training/testing/espresso/) tests in your pull request
 
 Finally:
 
-- it's good to start a new branch to start your work, not just for a shiny branch name, but also for your own good
+- as said, we have our day time work, except security vulnerabilities, your submission may be left cold for a couple of days before picked up by us
```

Comment:
"as said," would recommend removing this, it adds no useful information for the reader.

"before picked" missing verb, maybe "before being picked"?